Description: Save Debian user privacy
 The OpenStack theme is doing loads and loads of calls to random websites which
 could potentially (and most probably do) web browsing statistics.
 .
 This patch is an attempt to save Debian users from this ugliness.
 .
 Considering the amount of cruft I've seen, it's possible that I didn't remove
 it all. If you see some more, please let me know.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2017-09-09

Index: python-openstackdocstheme/openstackdocstheme/theme/openstackdocs/css.html
===================================================================
--- python-openstackdocstheme.orig/openstackdocstheme/theme/openstackdocs/css.html
+++ python-openstackdocstheme/openstackdocstheme/theme/openstackdocs/css.html
@@ -10,10 +10,3 @@
 <!-- Custom CSS -->
 <link href="{{pathto('_static/css/combined.css', 1)}}" rel="stylesheet">
 <link href="{{pathto('_static/css/styles.css', 1)}}" rel="stylesheet">
-
-<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
-<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-<!--[if lt IE 9]>
-    <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
-    <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
-<![endif]-->
Index: python-openstackdocstheme/openstackdocstheme/theme/openstackdocs/layout.html
===================================================================
--- python-openstackdocstheme.orig/openstackdocstheme/theme/openstackdocs/layout.html
+++ python-openstackdocstheme/openstackdocstheme/theme/openstackdocs/layout.html
@@ -16,19 +16,6 @@
 {% if favicon %}
     <link rel="shortcut icon" href="{{ pathto('_static/favicon.ico') }}"/>
 {% endif %}
-{% if theme_analytics_tracking_code %}
-<!-- Google Analytics -->
-<script>
-(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-ga('create', '{{ theme_analytics_tracking_code }}', 'auto');
-ga('send', 'pageview');
-</script>
-<!-- End Google Analytics -->
-{% endif %}
     <script type="text/javascript">
       var DOCUMENTATION_OPTIONS = {
         URL_ROOT:    '{{ url_root }}',
Index: python-openstackdocstheme/openstackdocstheme/theme/openstackdocs/script_search.html
===================================================================
--- python-openstackdocstheme.orig/openstackdocstheme/theme/openstackdocs/script_search.html
+++ python-openstackdocstheme/openstackdocstheme/theme/openstackdocs/script_search.html
@@ -1,3 +1,2 @@
-<script src="http://www.google.com/jsapi" type="text/javascript"></script>
 <script type="text/javascript">
 </script>
